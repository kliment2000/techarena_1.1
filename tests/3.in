3
500 10000 5000
6
1 a 50
1 i 500
2 s 450
2 r 450
3 f 475
3 s 475
1
1 a
3
1 2 i s
1 3 i f
2 3 r s
