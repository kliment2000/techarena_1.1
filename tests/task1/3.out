((1a 3 {1.i 3.f}) 2 {1.i 2.s} {3.s 2.r}) 38895.42105263158
